pigeon/notifications.dart

```diff
@@ -94,8 +94,39 @@ class MessagingStyle {
   final bool isGroupConversation;
 }
 
+/// Corresponds to `androidx.core.app.NotificationChannelCompat`
+///
+/// See: https://developer.android.com/reference/androidx/core/app/NotificationChannelCompat
+class NotificationChannel {
+  /// Corresponds to [`NotificationChannelCompat.Builder`](https://developer.android.com/reference/androidx/core/app/NotificationChannelCompat.Builder#Builder(java.lang.String,int))
```

Comment:
nit: one remaining instance of the pattern @chrisbobbe pointed out above

Reply:
Separately, the reference to the builder class is good but I think it's most accurate to link to the class as a whole rather than that constructor. Three of the five parameters here don't go to the builder's constructor, but rather to method calls on the builder.